osdep/threads.h

```diff
@@ -4,6 +4,8 @@
 #include <pthread.h>
 #include <inttypes.h>
 
+#include <config.h>
```

Comment:
```suggestion
#include "config.h"
```

Reply:
Ok, I generally include relative stuff as "" and something from other modules form `-I` as <>. But it is fine.